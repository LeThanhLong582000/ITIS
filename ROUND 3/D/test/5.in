6
685
341
726
69
907
90
