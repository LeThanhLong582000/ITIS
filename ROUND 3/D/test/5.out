5119
2293
5501
347
7057
463
